performance-test/scripts/SignUp_Test_Script.jmx

```diff
@@ -0,0 +1,1616 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.3">
```

Comment:
Sign Up Service - Reset Password (Preparation) sounds more like a setup

Reply:
@vishwa-vyom .. I have renamed to setup.